### PR TITLE
Fix test flake

### LINKED ISSUE #10053: Re-enable amp-bind integration tests on Saucelabs
```
Chrome 45.0.2454 (Windows 7 0.0.0) amp-bind with <amp-state> "before each" hook for "should not loop infinitely if updates change its src binding" FAILED
	Error: timeout of 10000ms exceeded. Ensure the done() callback is being called in this test.
..............................
Edge 15.15063.0 (Windows 10 0.0.0) amp-bind with <amp-list> "before each" hook for "should NOT change src when new value uses an invalid protocol" FAILED
	Error: timeout of 10000ms exceeded. Ensure the done() callback is being called in this test.
............................
Chrome 59.0.3071 (Windows 7 0.0.0) amp-bind with <amp-video> "before each" hook for "should change alt when the alt attribute binding changes" FAILED
	Error: timeout of 10000ms exceeded. Ensure the done() callback is being called in this test.
............................
Safari 9.1.2 (Mac OS X 10.11.6) amp-bind with <amp-iframe> "before each" hook for "should support binding to src" FAILED
	timeout of 10000ms exceeded. Ensure the done() callback is being called in this test.
Safari 9.1.2 (Mac OS X 10.11.6) amp-bind with <amp-list> "before each" hook for "should support binding to src" FAILED
	timeout of 10000ms exceeded. Ensure the done() callback is being called in this test.

### PR DESCRIPTION
**Pull request by jridgewell**
Fixes #10053.

### CODE REVIEW

File: test/integration/test-example-validation.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ if (!window.validatorLoad) {
   })();
 }
 
-describe.configure().retryOnSaucelabs().run('example', function() {
+describe.skip('example', function() {
```
Comment (aghassemi) [inline test/integration/test-example-validation.js]: Why skip all of these? Can we fix the failing example to validate or remove it from the list? If flaky on SauceLabs, I think a `.skipSauceLabs()` would be great to have especially for tests like this that don't really need cross-browser testing. ( as far as I remember, we run all tests including integration on Travis using latest Chrome in addition to running integration on SauceLabs, @rsimha-amp  is still still the case > )

Comment (jridgewell) [inline test/integration/test-example-validation.js]: If we skip sauce labs, when will these ever be run?

Comment (aghassemi) [inline test/integration/test-example-validation.js]: Just verified that `gulp test` (which runs on Travis with latest Chrome as part of the pre-submit and unit test job) does indeed run integration tests. (it runs `test/**/*`)

Comment (jridgewell) [inline test/integration/test-example-validation.js]: Done.

Comment (rsimha-amp) [inline test/integration/test-example-validation.js]: @aghassemi, [here](https://github.com/ampproject/amphtml/blob/master/build-system/pr-check.js#L253) is where the unit tests are run for PRs and on master. We used to run using old chrome on saucelabs, but no longer do.

I didn't think we ran integration tests with `gulp test`, unless the `--integration` arg was passed in, so your comment above comes as a surprise to me.

Comment (aghassemi) [inline test/integration/test-example-validation.js]: `gulp test` uses `testPaths` which is `test/**/*`. We definitely want to keep running integration tests on Travis with latest Chrome, so maybe the job can be renamed from `presubmit and unit tests` to something like `presubmit and all tests on latest Chrome` and the `integration tests` job can become `integrations tests on SauceLabs`

**Comment by jridgewell** [comment]
@aghassemi approval? 😛

File: test/integration/test-example-validation.js (hunk 1 above)
Comment (rsimha-amp) [inline test/integration/test-example-validation.js]: @aghassemi, fair enough. I'll review `pr-check.js` and do some clean up.

**Pull request merged**